service/pod/v2/resource/pod/create.go

```diff
@@ -4,6 +4,6 @@ import (
 	"context"
 )
 
-func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
```

Comment:
Going with the interface of the native resource here. Maybe I go back to the CRUD one later but for now should be good enough. Lets see.